testing/smoke/lib.sh

```diff
@@ -49,11 +72,12 @@ terraform_destroy() {
     if [[ ${exit_code} -gt 0 ]]; then
         echo "-> Smoke tests FAILED!!"
         echo "-> Printing terraform logs:"
-        cat tf.log
+	#cat tf.log
```

Comment:
Did you mean to leave this commented out?

Reply:
Yes, I find the previous behaviour really confusing.

Any error during the test assertions is getting suppressed by the tf.log printed after the test failed, so you end up having to scroll to find the error message.

I restored the previous behaviour and we can discuss this in a followup since it's unrelated to the changes.